src/transformer/descriptor/string/string.ts

```diff
@@ -1,5 +1,13 @@
 import * as ts from 'typescript';
+import { IsTsAutoMockRandomEnabled } from '../../../options/random';
+import { PropertySignatureCache } from '../property/cache';
+import { RandomPropertyAccessor } from '../random/random';
 
 export function GetStringDescriptor(): ts.Expression {
+  if (IsTsAutoMockRandomEnabled()) {
+    const propertyName: ts.PropertyName = PropertySignatureCache.instance.get();
+    const prefix: ts.StringLiteral = ts.createLiteral(`${(propertyName as ts.Identifier).escapedText}`);
```

Comment:
I'm probably wrong here but I think I remember we had a method to extract the string from a ts.PropertyName in a safe way (I feel like casting it as ts.Identifier won't work for every case). Is it something we actually have?